### DISCUSSION TITLE
How to detect droppable area when target container is empty

### DISCUSSION THREAD

**Discussion by sebastianservat** [clauderic/dnd-kit#688] [Q&A] [answered]
Hi @clauderic ,

I have extended the code you helped me with to achieve multiple sortable containers, but having 2 issues I need help with please:

https://codesandbox.io/s/dnd-kit-sortable-grid-multiple-nelb0c

1. Sometimes during the drag over action from one container to another and whilst trying to slot the item into place, I notice some performance degradation where the items slows down or even worst, gets stuck. Do you know why this might be happening and any improvement suggestions?

2. If I drag all items from one side to the other, I can't drag items back to the empty container as it doesn't detect a droppable area nor display the dragOverlay on the drop location. How can I handle this?

As usual, thanks a million for your support!

Seb.

**Comment by sebastianservat** [comment]
@clauderic Any suggestions?

**Comment by clauderic** [comment, marked as answer]
You need to connect this area to `useDroppable()`: 

<img width="640" alt="Screen Shot 2022-04-04 at 12 10 24 PM" src="https://user-images.githubusercontent.com/1416436/161586458-53c50aaa-8898-4658-95c8-2ac23ff299ca.png">

**Reply by sebastianservat** [reply]
Thank you! Implemented `useDroppable `and worked as expected :)
